### DISCUSSION TITLE
post/the-two-reacts

### DISCUSSION THREAD

**Discussion by giscus** [bepyan/bepyan.me.v2#9] [General]
# post/the-two-reacts

UI = f(data)(state)

https://bepyan.me/post/the-two-reacts

**Comment by omins** [comment]
잘 읽었습니다. 좋은 글 번역해주셔서 감사합니다 :)

**Reply by bepyan** [reply]
감사합니다. 도움이 되어 기쁘네요 :)

**Comment by Plush777** [comment]
블로그 이쁘네요! 참고하고 갑니다~  
(좋은 글도 감사합니다)

**Reply by bepyan** [reply]
좋게 봐주셔서 감사합니다 플러시님 :)
